viz.yaml

```diff
@@ -293,89 +293,84 @@ visualize:
       geom_sp_orig: "yahara_geom"
     title: Storm transposition
     alttext: Storm transposition
-publish:
-  -
-    id: publish_usgs_logo
-    location: images/usgsLogo.png
-    mimetype: image/png
-    title: "USGS Logo"
-    alttext: "USGS Logo"
+publish:  
+  - 
+    id: gdp_page
+    name: index
+    publisher: page
+    template: fullpage
+    depends: 
+      header: "header-section"
+      footer: "footer-section"
+      gdp_css: "gdp_figure_css"
+      header_css: "lib-header-css"
+      footer_css: "lib-footer-css"
+      font: "sourcesanspro"
+      text_overview: "gdp_overview"
+      section_process: "gdp_processing_section"
+      section_algorithm: "gdp_algorithm_section"
+      text_usecase: "use_case_intro"
+      section_storm_transpose: "storm_transpose_section"
+      #section_data_input: "inputs_data_section"
+      section_geometry_input: "inputs_geometry_section"
+      #section_clip_melt: "yahara_clip_process_section"
+      text_conclusion: "conclusion_text"
```

Comment:
This is fine, but I believe, we don't have to do this kind of naming anymore if you don't want to - walker made a change a while back so dependencies automatically get the are named the same as the yaml section if they aren't assigned one.  The way you have it here is still useful though if you want to be able to swap out yaml sections without changing the script.

Reply:
Yeah, that's how I had it but I followed the [example viz](https://github.com/USGS-VIZLAB/example) and changed it because I think this is more readable